Add circleci

cxm6467 opened the pull request:
+ circleci

cxm6467 committed:
Add circleci

KyleDuca committed:
Merge branch 'master' into add_circleci

KyleDuca commented:
Nope, getting this:

#!/bin/bash -eo pipefail
bundle exec rake db:create
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5432?

Try autoconfig or I'll try to fix it later.

cxm6467 commented:
I’ve had that issue on my personal site. Only way I fixed it was to remove
what I added then add it back but I don’t think that’s a proper fix.

On Thu, Nov 9, 2017 at 3:52 PM Kyle Duca <notifications@github.com> wrote:

> Nope, getting this:
>
> #!/bin/bash -eo pipefail
> bundle exec rake db:create
> could not connect to server: Connection refused
> Is the server running on host "localhost" (127.0.0.1) and accepting
> TCP/IP connections on port 5432?
>
> Try autoconfig or I'll try to fix it later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sudo-chess/chess-app/pull/5#issuecomment-343287786>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABRbQiDIChh75lS6TmaBd_umhf8Qy_Erks5s02YqgaJpZM4QYhzH>
> .
>
-- 
Regards, Chris Marasco RIT '12 | BS Information Technology Kaplan'15 | MS
Information Security and Assurance A+, VCA-DCV, VCA-Cloud Certified
